specs/answers.spec.ts

```diff
@@ -0,0 +1,70 @@
+import algolia from "../packages/algoliasearch/dist/algoliasearch";
+declare const algoliasearch: typeof algolia;
+
+declare const browser: {
+  executeAsync<TArg, TResult>(
+    cb: (arg: TArg, done: (res: TResult) => TResult) => void,
+    arg: TArg
+  ): TResult;
+
+  url(to: string): void;
+};
+
+const credentials = {
+  appId: "CKOEQ4XGMU",
+  apiKey: "6560d3886292a5aec86d63b9a2cba447"
+  // TODO: change these credentials to the main ones once enabled on our app
+  // appId: `${process.env.ALGOLIA_APPLICATION_ID_1}`,
+  // apiKey: `${process.env.ALGOLIA_SEARCH_KEY_1}`
+};
+
+describe("answers features - algoliasearch.com", () => {
+  beforeEach(async () => browser.url("algoliasearch.com"));
+
+  it("searchIndex::findAnswers", async () => {
+    const results: any = await browser.executeAsync(function(
+      credentials,
+      done
+    ) {
+      const client = algoliasearch(credentials.appId, credentials.apiKey);
+
+      // TODO: remove this customization once the engine accepts url encoded query params
+      client.transporter.userAgent.value = "answers-test";
```

Comment:
@Haroenv to be clear, are we waiting for the engine fix, so that we can get rid of this line before merging this PR?

Reply:
Hey @eunjae-lee, the engine fix won't be happening soon, so I'd rather merge this and prepare a v1.1 PR for when the engine has catched up :)

Reply:
got it. thanks for clarification!

Reply:
FYI, [issue opened here](https://algolia.atlassian.net/browse/SRCH-1773) :)